<ui version="4.0" >
 <class>CClientSettingsDlgBase</class>
 <widget class="QDialog" name="CClientSettingsDlgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>General Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/png/main/res/mainicon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGroupBox" name="GroupBoxJitterBuffer" >
     <property name="title" >
      <string>Jitter Buffer</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbAutoJitBuf" >
        <property name="text" >
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TextNetBuf" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Size</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="SliderNetBuf" >
          <property name="pageStep" >
           <number>1</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition" >
           <enum>QSlider::TicksBothSides</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="CMultiColorLED" native="1" name="CLEDNetw" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBoxSoundCard" >
     <property name="title" >
      <string>Soundcard</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="TextSoundcardDevice" >
        <property name="text" >
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbSoundcard" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>153</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="FrameSoundcardChannelSelection" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbInChannelMapping" >
           <property name="text" >
            <string>Input Channel Mapping</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>3</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QLabel" name="labelLInChan" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>L</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRInChan" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>R</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>3</number>
             </property>
             <item>
              <widget class="QComboBox" name="cbLInChan" />
             </item>
             <item>
              <widget class="QComboBox" name="cbRInChan" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbOutChannelMapping" >
           <property name="text" >
            <string>Output Channel Mapping</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="spacing" >
            <number>3</number>
           </property>
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QLabel" name="labelLOutChan" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>L</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelROutChan" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text" >
                <string>R</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>3</number>
             </property>
             <item>
              <widget class="QComboBox" name="cbLOutChan" />
             </item>
             <item>
              <widget class="QComboBox" name="cbROutChan" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>153</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="GroupBoxSoundCrdBufDelay" >
        <property name="title" >
         <string>Buffer Delay</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QRadioButton" name="rButBufferDelayPreferred" >
           <property name="text" >
            <string>(preferred)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rButBufferDelayDefault" >
           <property name="text" >
            <string>(default)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rButBufferDelaySafe" >
           <property name="text" >
            <string>(safe)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonDriverSetup" >
           <property name="text" >
            <string>Driver Setup</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBoxMeasureResults" >
     <property name="title" >
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbOpenChatOnNewMessage" >
        <property name="text" >
         <string>Open Chat on New Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbGUIDesignFancy" >
        <property name="text" >
         <string>Fancy GUI Design</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbUseHighQualityAudio" >
        <property name="windowModality" >
         <enum>Qt::NonModal</enum>
        </property>
        <property name="text" >
         <string>Use High Quality Audio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbUseStereo" >
        <property name="windowModality" >
         <enum>Qt::NonModal</enum>
        </property>
        <property name="text" >
         <string>Stereo Streaming</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="LabelCentralServerAddress" >
          <property name="text" >
           <string>Central Server Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbDefaultCentralServer" >
          <property name="text" >
           <string>Default</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="LineEditCentralServerAddress" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>201</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="TextPingTime" >
          <property name="text" >
           <string>Ping Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabelPingTime" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>val</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="TextOverallDelay" >
          <property name="text" >
           <string>Overall Delay</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabelOverallDelay" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>val</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CMultiColorLED" native="1" name="CLEDOverallDelay" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="TextUpstream" >
          <property name="text" >
           <string>Audio Stream Rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextUpstreamValue" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>val</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMultiColorLED</class>
   <extends>QWidget</extends>
   <header>multicolorled.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbAutoJitBuf</tabstop>
  <tabstop>SliderNetBuf</tabstop>
  <tabstop>cbSoundcard</tabstop>
  <tabstop>cbLInChan</tabstop>
  <tabstop>cbRInChan</tabstop>
  <tabstop>cbLOutChan</tabstop>
  <tabstop>cbROutChan</tabstop>
  <tabstop>rButBufferDelayPreferred</tabstop>
  <tabstop>rButBufferDelayDefault</tabstop>
  <tabstop>rButBufferDelaySafe</tabstop>
  <tabstop>ButtonDriverSetup</tabstop>
  <tabstop>cbOpenChatOnNewMessage</tabstop>
  <tabstop>cbGUIDesignFancy</tabstop>
  <tabstop>cbUseHighQualityAudio</tabstop>
  <tabstop>cbUseStereo</tabstop>
  <tabstop>cbDefaultCentralServer</tabstop>
  <tabstop>LineEditCentralServerAddress</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
